fuzz/fuzz_targets/fuzz_expr.rs

```diff
@@ -21,7 +21,9 @@ static CMD_PATH: &str = "expr";
 
 fn generate_expr(max_depth: u32) -> String {
     let mut rng = rand::thread_rng();
-    let ops = ["+", "-", "*", "/", "%", "<", ">", "=", "&", "|"];
+    let ops = [
+        "+", "-", "*", "/", "%", "<", ">", "=", "&", "|", "!=", "<=", ">=", ":", "length", "substr",
```

Comment:
```suggestion
        "+", "-", "*", "/", "%", "<", ">", "=", "&", "|", "!=", "<=", ">=", ":", "index", "length", "substr",
```

Reply:
yeah, i removed it because it was failing immediately 
but now that we have the expected fails in the CI, we should do it indeed. thanks :)